src/plist.c

```diff
@@ -2542,65 +2542,65 @@ static Obj FuncIsRectangularTablePlist(Obj self, Obj plist)
 *V  BagNames  . . . . . . . . . . . . . . . . . . . . . . . list of bag names
 */
 static StructBagNames BagNames[] = {
-  { T_PLIST,                                "list (plain)" },
-  { T_PLIST            +IMMUTABLE,          "list (plain,imm)" },
+  { T_PLIST,                                "plain list" },
+  { T_PLIST            +IMMUTABLE,          "immutable plain list" },
```

Comment:
Not very important, but: I kinda disagree with the commit message / title which refers to "more human-readable names". To me, they are equally readable; the difference is in how "understandable" / "intelligible" / "comprehensible" they are.